101-hdinsight-spark-kafka/README.md

```diff
@@ -0,0 +1,10 @@
+# Deploy an Azure Virtual Network and a Kafka on HDInsight cluster within the network
```

Comment:
Change to: Kafka on HDInsight

remove the part about creating a vnet since all clusters are in a vnet

Reply:
@bmoore-msft Correct, each HDInsight by default is in it's own virtual network. Unfortunately, there's no way to communicate between the clusters in this configuration other than the default gateway (https://clustername.hdinsight.net). Even more unfortunately, Kafka APIs are not exposed over this public gateway.

In order for *anything* to work with Kafka on HDInsight, it either needs to be on the same cluster, or joined to the same virtual network. We cannot change or add things to the auto-created virtual network HDInsight creates. So the only solution currently is to create a normal virtual network, then create Kafka and anything that communicates with Kafka, inside that virtual network.

Reply:
Change the title of the readme to:

Deploy a Kafka on HDInsight Cluster within a Virtual Network